chain/src/main/scala/org/bitcoins/chain/models/BlockHeaderDAO.scala

```diff
@@ -0,0 +1,94 @@
+package org.bitcoins.chain.models
+
+import org.bitcoins.chain.config.ChainConfig
+import org.bitcoins.core.crypto.DoubleSha256DigestBE
+import org.bitcoins.core.protocol.blockchain.ChainParams
+import org.bitcoins.db.{CRUD, DbConfig, SlickUtil}
+import slick.jdbc.SQLiteProfile.api._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Created by chris on 9/8/16.
```

Comment:
Scaladoc is outdated
Probably remove line 12, and either change Postgresql to Sqlite on line 14 or just remove sentence?